raiden/transfer/mediated_transfer/state.py

```diff
@@ -12,6 +14,10 @@
 )
 from raiden.utils import pex, serialization, sha3, typing
 
+# Upgrade pyflakes to 2.0.0 and remove the 'if' and '# noqa'.
+if TYPE_CHECKING:
+    from raiden.transfer.mediated_transfer.events import SendSecretReveal  # noqa: F401
```

Comment:
question: So pyflakes complains that `SendSecretReveal` is not used anywhere but you only use it for type checking? That's why this is here?

Reply:
* mypy wants to parse this string literal `'SendSecretReveal'` as a Python symbol.  So mypy needs the import.
* pyflake doesn't parse the string literal `'SendSecretReveal'` because it is just a string literal.  So pyflake doesn't like the import.

https://github.com/raiden-network/raiden/blob/bb27bdb540b24fa49b93cc87d1ea8a978149a597/raiden/transfer/mediated_transfer/state.py#L108> typing.Optional['SendSecretReveal'],